tests/acceptance/stepDefinitions/previewContext.js

```diff
@@ -0,0 +1,13 @@
+const { client } = require('nightwatch-api')
+const { When, Then } = require('cucumber')
+const offCanvasPage = client.page.FilesPageElement.offCanvasPage()
+const filesList = client.page.FilesPageElement.filesList()
+
+When('user views the file {string} in the mediaviewer using the webUI', async function(fileName) {
+  await filesList.clickOnFileName(fileName)
+  return offCanvasPage.openMediaViewer()
+})
+
+Then('the file {string} should be opened in the mediaViewer', function(fileName) {
```

Comment:
```suggestion
Then('the file {string} should be displayed in the mediaViewer', function(fileName) {
```

Maybe this is a better word to use?